src/base/base.proto

```diff
@@ -49,6 +49,9 @@ message Flags {
   repeated string included_files = 12;
   // PCH or PTH files, included implicitly through command line arguments.
 
+  optional bool rewrite_includes = 13;
+  // Indicates if preprocessing should/was made using frewrite-includes flag.
```

Comment:
Flag is passed without dash to cc1 or driver, so I intentionally skipped it every time it is mentioned. I'll take it in quotes, no problem here.